RadClass/RadClass.py

```diff
@@ -95,16 +94,14 @@ def queue_file(self):
         if self.start_time is not None:
             timestamp = self.processor.timestamps[self.processor.timestamps >=
                                                   self.start_time][0]
-            self.start_i = np.where(self.processor.timestamps ==
-                                    timestamp)[0][0]
+            self.start_i = np.abs(self.processor.timestamps - timestamp).argmin()
```

Comment:
I saw [here](https://stackoverflow.com/questions/16243955/numpy-first-occurrence-of-value-greater-than-existing-value) a discussion about using `np.searchsorted` since the timestamps array is instrinsically sorted (we also discussed this in S/W). This new method should work, choosing either where `self.processor.timestamps = timestamp` or the first instance less than (or the first value in the array if `timestamp` is less than all values in the array).